src/redux/categories/categories.js

```diff
@@ -0,0 +1,24 @@
+// Define constants
+const CHECKSTATUS = 'CHECK_STATUS';
```

Comment:
- [ ] Great job on all the functionality but here I think it would be great if you Define action types for adding and removing a book. in DUCK pattern so I recommend this to be somthing like this you can check this [article](https://github.com/erikras/ducks-modular-redux) for more info ❤✅🔥✨

const CHECK_STATUS = bookstore/categories/CHECK_STATUS